omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_checks.rb

```diff
@@ -44,7 +44,7 @@ def first_run?
   end
 
   def secrets_exists?
-    PrivateChef.existing_secrets.length > 0
+    File.exist?("/etc/opscode/private-chef-secrets.json")
```

Comment:
I changed it initially because `existing_secrets` no longer exists with this implementation. If we think that having an invalid file is reason to stop then I can definitely load the file and ensure that the secrets count is non-zero.